### PR TITLE
KOJO-21 | Remove satis dependency from kōjō composer.json

### PR DESCRIPTION
**Pull request by p-mucha**
Running `composer update` after making these modifications showed no significant changes in `composer.lock` so I didn't update that, to reduce the likelihood of something dependency update breaking things, but I can include it if you guys think otherwise

### CODE REVIEW

**Comment by p-mucha** [comment]
I was thinking exactly that, especially when I saw the example is still set to use `^3.0.0`

@rhift Any objections to me removing `example/` and updating the `README`?

**Comment by rhift** [comment]
> @rhift Any objections to me removing example/ and updating the README?

aggressively refactor with fury. purge that artifact with passion.

File: composer.json [hunk 1]
```diff
@@ -8,17 +8,19 @@
         {
             "name": "Brad Wilson",
             "email": "brad.wilson@neighborhoods.com"
+        },
+        {
+            "name": "Alex Berryman",
+            "email": "alex.berryman@neighborhoods.com"
+        },
+        {
+            "name": "Przemyslaw Mucha",
```
Comment (rhift) [inline composer.json]: 👍

File: composer.json [hunk 2]
```diff
@@ -8,17 +8,19 @@
         {
             "name": "Brad Wilson",
             "email": "brad.wilson@neighborhoods.com"
+        },
+        {
+            "name": "Alex Berryman",
```
Comment (rhift) [inline composer.json]: 👍

**Pull request merged**